app/Http/Controllers/LoginController1.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace App\Http\Controllers;
+use \App\User;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Hash;
+
+class LoginController1 extends Controller
+{
+   public function index() {
+    return view('login');
+   }
```

Comment:
This review goes for all your controller methods. You're used to writing functions where the opening `{` is on the same line as the function declaration. In php, Object Oriented programming in general, the opening `{` is always _on the next line_. So:
```php
public function index() 
{
  // do stuff here
}
```

When you gen a controller with artisan, you can see this pattern. I'd keep everything within this convention.

Reply:
got it. Will make the changes and push.